README.md

```diff
@@ -212,6 +221,55 @@ docker run -p 6379:6379 envirodgi/db-import-worker -e <ENVIRONMENT VARIABLES> .
+
+There are several other kinds of objects, but they are subservient to the ones above:
+
+- **Changes**, which serve to connect any two *versions* of a *page*. *Annotations* are actually connected to *changes*, rather than directly to two *versions*. You can also generate diffs for a given *change*.
+
+- **Tags**, which can be applied to pages. They help sort and categorize things. Most tags are manually applied, but the application auto-generates a few:
+    - `domain:<domain name>`, e.g. `domain:www.epa.gov` for a page at `https://www.epa.gov/citizen-science`
+    - `2l-domain:<second-level domain name>` e.g. `2l-domain:epa.gov` for a page at `https://www.epa.gov/citizen-science`
+
+- **Maintainers**, which can be applied to pages. They represent organizations that maintain a given page. For example, the page at `https://www.epa.gov/citizen-science` is maintained by `EPA`.
+
+- **Imports** model requests to import new data and the results of the import operation.
+
+- **Users** model people (both human and bots) who can view, import, and annotate data. You currently have to have a user account to do anything in  the application, though we hope accounts will not be needed to view public data in the future.
+
+
+### How Data Gets Loaded
+
+The web-monitoring-db project does not actually monitor or scrape actual pages on the web. Instead, we rely on importing data from other services, like [the Internet Archive](https://archive.org). Each day, a script queries other services for historical snapshots and sends the results to the `/api/v0/imports` endpoint.
+
+Most of the data sent to `/api/v0/imports` matches up directly with the structure of the [`Version` model](https://github.com/edgi-govdata-archiving/web-monitoring-db/blob/master/app/models/version.rb). However, the `uri` field in an import is treated specially. If the `uri` host matches one of the hosts listed in the [`ALLOWED_ARCHIVE_HOSTS` environment variable](https://github.com/edgi-govdata-archiving/web-monitoring-db/blob/master/.env.example), the application simply stores that as the version’s `uri`. If it doesn’t match, the application downloads the content from `uri` and stores it in its `FileStorage`. The intent is to make sure data winds up at a reliably available location, ensuring that anyone who can access the API can also access the raw response body for any version. The application’s storage area can be the local disk or it can be S3, depending on configuration. The component can take pluggable configurations, so we can support other storage types or locations in the future.
```

Comment:
The description about `uri` and what is stored is confusing here. When it says "the application simply stores that as the version's `uri`", I'm not sure what "that" is. It makes me think that some blob is stored in `uri` (which I'm assuming should be a string, but now I'm not sure, so I'm lost). 

When the hosts match, nothing extra is stored and the app just grabs from the archive host? Is that correct? If so, using the term "stored" may also be why it sounds confusing.

The explanation in the comments in `.env.example` are actually really clear! If we can transfer some of that language to this section, I feel it would be more understandable. 

I pulled some of the comments into the suggestion below as an example, but I'm not sure about verbiage and left some question marks. Please don't take the suggestion as-is!
```suggestion
Most of the data sent to `/api/v0/imports` matches up directly with the structure of the [`Version` model](https://github.com/edgi-govdata-archiving/web-monitoring-db/blob/master/app/models/version.rb). However, the `uri` field in an import is treated specially. 

When new page or version data is imported, the `uri` field points to a location where the raw HTTP response body is stored. If the `uri` host does *not* match one of the values in `ALLOWED_ARCHIVE_HOSTS`, the application downloads the data from `uri` and stores it in its `FileStorage`. If the `uri` host matches one of the hosts listed in the [`ALLOWED_ARCHIVE_HOSTS` environment variable](https://github.com/edgi-govdata-archiving/web-monitoring-db/blob/master/.env.example), the application simply [points to]?? [what was given]?? as the version’s `uri`. 

The intent is to make sure data winds up at a reliably available location, ensuring that anyone who can access the API can also access the raw response body for any version. The application’s storage area can be the local disk or it can be S3, depending on configuration. The component can take pluggable configurations, so we can support other storage types or locations in the future.
```

Reply:
> When the hosts match, nothing extra is stored and the app just grabs from the archive host?

If they match, the app grabs nothing. It’ll just point to that external location whenever someone wants the raw response data for that version record.

Reply:
Ok, took another crack at this in 712782c91740d57118fc67ec0514ea9e0bd2e021. How does that read? https://github.com/edgi-govdata-archiving/web-monitoring-db/blob/249-469-explain-just-who-gets-trusted-to-hold-your-gooses-golden-eggs-and-where/README.md#how-data-gets-loaded

Reply:
Looks good!